pkg/kubectl/cmd/logs/logs.go

```diff
@@ -153,6 +154,7 @@ func NewCmdLogs(f cmdutil.Factory, streams genericclioptions.IOStreams) *cobra.C
 	cmd.Flags().Int64Var(&o.LimitBytes, "limit-bytes", o.LimitBytes, "Maximum bytes of logs to return. Defaults to no limit.")
 	cmd.Flags().BoolVarP(&o.Previous, "previous", "p", o.Previous, "If true, print the logs for the previous instance of the container in a pod if it exists.")
 	cmd.Flags().Int64Var(&o.Tail, "tail", o.Tail, "Lines of recent log file to display. Defaults to -1 with no selector, showing all log lines otherwise 10, if a selector is provided.")
+	cmd.Flags().BoolVar(&o.AllowPodErrors, "allow-errors", o.AllowPodErrors, "If watching / following pod logs, allow for any errors that occur to be non-fatal")
```

Comment:
Nit: I think `ignore-errors` is more appropriate and in-line with other `ignore-not-found` flags.